common/constants.go

```diff
@@ -231,3 +229,33 @@ const (
 	RecordClosed           VisibilityOperation = "RecordClosed"
 	UpsertSearchAttributes VisibilityOperation = "UpsertSearchAttributes"
 )
+
+const (
+	numBitsPerLevel = 3
+
+	// NoPriority is the value returned if no priority is ever assigned to the task
+	NoPriority = -1
+)
+
+const (
+	// HighPriorityClass is the priority class for high priority tasks
+	HighPriorityClass = iota << numBitsPerLevel
```

Comment:
I'm fuzzy on why these had to move, but also I don't really mind either way.  just curious / forgetful.

Reply:
common/task package imports common/dynamicconfig package.
we had to move it to avoid circular dependency

Reply:
ah, due to combining the types / keys / etc probably.  yea, that's fine.